src/graphql/main/schema.graphql

```diff
@@ -877,6 +877,7 @@ scalar PaymentHash
 
 type PaymentSendPayload {
   errors: [Error!]!
+  transaction: Transaction
```

Comment:
It would be nice if PaymentSendPayload returned a full transaction object, that way the transaction enters the cache and we can add it to our list of transactions without having to requery transactions from the home screen. It also allows us to link directly to the transaction details in the application. Currently we dont have a transaction id returned from the mutation so we cant link within the app to the correct page.

Reply:
here, transaction probably optional